test/suite/test_timestamp03.py

```diff
@@ -254,16 +334,56 @@ def test_timestamp03(self):
             count += 1
         # print "Updated " + str(count) + " keys to value3"
 
+        # Scenario : 6
+        # when read time stamp != stable timestamp, take checkpoints using both
+        # stable timestamp and read timestamp and check to see the values differ
```

Comment:
"time stamp" should be "timestamp".  Add 'the' before using them in a sentence too.